daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -363,10 +363,9 @@ class Engine(val config: EngineConfig = new EngineConfig(LanguageVersion.StableV
           nodeSeeds = onLedger.ptx.nodeSeeds.toImmArray,
         )
         config.profileDir.foreach { dir =>
-          val hash = meta.nodeSeeds(0)._2.toHexString
           val desc = Engine.profileDesc(tx)
-          machine.profile.name = s"$desc-${hash.substring(0, 6)}"
-          val profileFile = dir.resolve(s"${meta.submissionTime}-$desc-$hash.json")
+          machine.profile.name = s"${meta.submissionTime}-$desc"
+          val profileFile = dir.resolve(s"${meta.submissionTime}-$desc.json")
```

Comment:
This change is deliberate. Including the transaction seed makes no sense for users.